docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+// To do: Add a page that talks about how synthetics and UE work together. Passive/Active, etc.
+
+[[user-experience]]
+= {user-experience}
```

Comment:
Should this title focus on a user goal, similar to what we have for the logs/metrics/uptime topics? Something like Analyze user experience or Monitor user experience?

Reply:
Looks like this has been addressed in this [PR](https://github.com/elastic/observability-docs/pull/198).